Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -21,6 +21,10 @@ class PostRideSummaryViewController: UIViewController {
     
     private var ride: Ride!
     
+    // MARK: Networking
+    private let networkManager = NetworkManager()
```

Comment:
Is there a reason that you're saving network manager in a let variable like this? Most functions and fields in NetworkManager can be accessed through NetworkManager.shared.

Reply:
Good catch, didn't realize I should have been calling the singleton 🫡